correct various typos

ReblochonMasque opened the pull request:
Several minor typos were corrected

swt2c commented on docs/usage/usage_advanced_debug.rst:
```diff
@@ -64,11 +64,11 @@ not be sent the message. Putting a try/except clause around every sendMessage is
 
 Since exceptions are common during application development (bugs due to 
-invalid arguments, failed assertions, etc), Pypubsub provdes a hook to register
+invalid arguments, failed assertions, etc...), Pypubsub provides a hook to register
```
I would argue there should just be a single period here after the etc.  An ellipsis (...) is somewhat redundant with etc.

swt2c commented on docs/usage/usage_advanced_debug.rst:
```diff
@@ -64,11 +64,11 @@ not be sent the message. Putting a try/except clause around every sendMessage is
 Pypubsub then sends it to the handler, and continues with the send operation
 until all listeners have received the message. The handler might print it to
-a log file, output a message in a status bar, show an error box, etc. The 
+a log file, output a message in a status bar, show an error box, etc... The 
```
Same here, I would not add an ellipsis.

swt2c commented on docs/usage/usage_advanced_debug.rst:
```diff
@@ -78,14 +78,14 @@ of the handler can be given to ``pub.setListenerExcHandler()``.
 However, any listener that is a callable
 with a "catch-all" \**kwargs parameter will be given all message data. Moreover, 
-Pypubsub sends the topic object automatically with the message data if it
+Pypubsub sends the topic object automatically with the message data, if it
```
I don't think a comma is appropriate here.

ReblochonMasque commented:
Noted: I noticed the missing chars, corrected them, and thought that I'd scan the entire page for other things to correct in order to do a thorough job, and I think I got carried away trying to find stuff, sorry.
I reverted the three changes you pointed at.

schollii commented:
Thanks I will check this once I get back from vacation next week

schollii commented:
Thanks @ReblochonMasque for fixes and @swt2c for reviewing, much appreciated!
